### DISCUSSION TITLE
selene choice like upgrades in store

### DISCUSSION THREAD

**Discussion by quiloulou** [Malcolm-Q/LC-LateGameUpgrades#498] [Ideas]
hi, i love this mod but i think the high amount of upgrades makes me hesitant on which upgrade to choose
why not adding an option that choose 3 upgrades each shift and add them to the terminal store like selene's choices for the planets but for the latesgames upgrades (the command lategame store is desactivated)
the tiers upgrades can be buy only in the store choice and they they enter the pool when the upgrade is purchased.
example: I buy the lvl 1 upgrade, so lvl 2 can appear in the store
i'm french sorry for the bad translate

**Comment by WhiteSpike** [comment]
Closing as implemented in 3.9.1 release
